config/externally_supplied_metadata/README.md

```diff
@@ -0,0 +1,32 @@
+## Externally supplied metadata
+#### E Flynn
+#### Last updated 3/25/2020
+
+All code to produce this update is included in the [sl_label repository](https://github.com/erflynn/sl_label repository)
+
+This update includes imputed sex labels for microarray data (mouse, rat, and human).
+
+The majority of gene expression data is missing metadata sex labels (see Table 1). This lack of labels prevents us from examining the breakdown by sex of many studies. We used the expression of X and Y chromosome genes and metadata sex labels to train a logistic regression model (with elastic net penalty) to predict sample sex. Across all three organisms, our models achieve approximately 95% accuracy in a randomly selected held-out test set as compared to the metadata labels. Additionally, we assessed the accuracy of our model, on various subsets of the data; comparing to all metadata sex labels (agreement 93.5-94.8%), a random sample of single sex studies (agreement 92.6-96.5%), and, in human, manually annotated sex labels from a previous analysis (94.2%) (see Table 2).  
+
+| organism | Samples (n) | Samples missing metadata annotation | Studies (n) | Studies missing metadata annotation |
+| ----- | ---- | ---- | ---- | ---- |
+| human | 430119 | 74.90% | 14987 | 87.30% |
```

Comment:
Re the csv files - you are right, there are duplicates, apologies here! I have updated these, and put in a commit that fixes that. The numbers now exactly match those in the table.

For the run accessions, these are exactly the set of accessions that are included in the `aggregated_metadata.json` files in the respective compendia (the normalized microarray and then the RNA-seq compendia). You are right that there is some overlap -- I will look into this more -- but the accessions are the same. Is there a different way that you usually organize this?

Reply:
> For the run accessions, these are exactly the set of accessions that are included in the `aggregated_metadata.json` files in the respective compendia (the normalized microarray and then the RNA-seq compendia). You are right that there is some overlap -- I will look into this more -- but the accessions are the same.

Ah okay I see! Then I would expect some overlap because the normalized compendium contains both microarray and RNA-seq data. So maybe it would be better to call these files `normalized` and `rnaseq_sample` rather than `microarray` and `rnaseq` to match the terminology here: https://www.refine.bio/compendia.

Reply:
ah ok! I will update this, thank you!